resources/services/lightsail/disks.go

```diff
@@ -0,0 +1,321 @@
+package lightsail
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/lightsail"
+	"github.com/aws/aws-sdk-go-v2/service/lightsail/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource disks --config gen.hcl --output .
+func Disks() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_lightsail_disks",
+		Description:  "Describes a block storage disk",
+		Resolver:     fetchLightsailDisks,
+		Multiplex:    client.ServiceAccountRegionMultiplexer("lightsail"),
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"arn"}},
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the disk",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "attached_to",
+				Description: "The resources to which the disk is attached",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "attachment_state",
+				Description: "(Deprecated) The attachment state of the disk",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "created_at",
+				Description: "The date when the disk was created",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:          "gb_in_use",
+				Description:   "(Deprecated) The number of GB in use by the disk",
```

Comment:
Should this column be included at all if it's deprecated?

Reply:
I would say that as long as the API is returning the data we should persist it and allow users to access it

Reply:
Is the API returning the data though?

Reply:
Deprecated in AWS typically means that you can't create a new resource with the parameters but if you have existing resources using it will continue to work

Reply:
Btw, I'm okay either way and don't think this should block merging. 

But I guess if the API doesn't return anything, one day we may get an issue from someone who didn't see the column description and realize it was a deprecated field.